### DISCUSSION TITLE
installazione MQTT su Home Assistant OS/Supervised

### DISCUSSION THREAD

**Discussion by domantonio3001** [indomus/forum#3031] [HUB: Home Assistant]
ciao a tutti,
scusate sono un neofita sto provando ad installare  Home Assistant OS/Supervised su PI4 seguendo pedissequamente la guida proposta da inDomus, purtroppo arrivato alla installazione di MQTT al momento di salvare la configurazione il sistema mi risponde che non può salvare la configurazione con la seguente segnalazione:

Impossibile salvare la configurazione del componente aggiuntivo, Missing option 'logins' in root in Mosquitto broker (core_mosquitto). Got {'require_certificate': False, 'certfile': 'fullchain.pem', 'keyfile': 'privkey.pem', 'customize': {'active': False, 'folder': 'mosquitto'}}

Ringrazio chi può venirmi in aiuto
antonio

**Comment by snakuzzo** [comment]
![Screenshot_20240410-200907](https://github.com/indomus/forum/assets/7205371/e366f4c0-e111-41b5-b661-d9e95256caea)

**Comment by domantonio3001** [comment]
ok fatto grazie
vado al passo successivo
sbalordito per la velocità di risposta
ciao
